### ISSUE TITLE
[a11y] "My devices" screen content is not read while using a screen reader

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7699]
**Important details:**

* All content of the My devices screen is read as “Custom”;

**Mozilla VPN affected versions:**

* 2.17.0 (2.202308070805);

**Platform tested:**

* Windows 10;
* MacOS 13;

**Prerequisites:**

* Have a fresh install of the Mozilla VPN;
* Have an account with subscription for Mozilla VPN;
* Be logged in;
* Have a screen reader turned on;

**Steps to reproduce:**

1. Using the keyboard navigate to Settings - My Devices;
1. Observe the content read;

**Expected results:**

* The Narrator/Voice over tools read the entire content of the screen.

**Actual results:**

* The screen reader does not read the screen content;

**Notes:**

* The devices are read while in Edit mode; 
* the “x of 5 devices added” text is not read even if in Edit mode;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5326)
┆Reporter: Valentina Virlics

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Same applies for “6th device screen“.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Rechecked this on both platforms, as follows:

  * Windows: caps+O => content of the My devices screen is read correctly;
  * MacOS: Quick nav on => content of the My devices screen is read correctly;

We can close this as “works-for-me”;

Thank you for the tips, Matt Lichtenstein !

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Cancelling.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
